README.md

```diff
@@ -440,6 +440,31 @@ Do not leave console calls in your code
 
 > Reason: it just a noise for other developers. We always can find anything using history of version control system.
 
+### Specify (`TODO` | `FIXME`) comment with JIRA (or another bug tracking system) task id or username
```

Comment:
Can we simplify it a little bit, say

### Always specify (`TODO` | `FIXME`) comment with task id or username